sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -1,23 +1,27 @@
 package com.klaviyo.coresdk
 
 import com.klaviyo.coresdk.networking.KlaviyoEvent
+import com.klaviyo.coresdk.networking.UserInfo
 import com.klaviyo.coresdk.networking.requests.IdentifyRequest
 import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
 import com.klaviyo.coresdk.networking.requests.TrackRequest
 
 object Klaviyo {
-    fun track(event: KlaviyoEvent, customerProperties: Map<String, String>, properties: Map<String, String>? = null) {
+    fun setUserEmail(email: String) {
+        UserInfo.email = email
+    }
+
+    fun track(event: KlaviyoEvent, customerProperties: Map<String, Any>, properties: Map<String, Any>? = null) {
         val request = TrackRequest(event.name, customerProperties.toMutableMap(), properties)
         request.generateUnixTimestamp()
         processRequest(request)
     }
 
-    fun identify(properties: Map<String, String>) {
+    fun identify(properties: Map<String, Any>) {
```

Comment:
This needed to be updated to `Any` to hold onto nested maps (needed for adding push tokens in the append props).  We'll be changing this API a bit later so its less free-form but for the time being we just need the types to comply